source/components/inputs/select/select.ng1.ts

```diff
@@ -96,10 +96,8 @@ export class SelectController extends InputController {
 		this.showOptions = !this.showOptions;
 	}
 
-	close: { (): void } = () => {
-		if (this.showOptions) {
```

Comment:
It looks like this more or less just happened because you were trying different things so when you came back to this it was different than when you'd started...?